ios/RNCNetInfo.h

```diff
@@ -7,6 +7,10 @@
 
 #import <React/RCTEventEmitter.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
This appears to be a convenience macro that simply avoids specifying things are nonnull directly
https://stackoverflow.com/questions/37956272/ns-assume-nonnull-begin-macro

Is that right?

It does not seem necessary here, and either way with our really small API surface area I'd prefer they were specified directly vs as a side effect of a macro, so this should be removed